lib/extensions/src/lib/components/dynamic-component/dynamic.component.spec.ts

```diff
@@ -72,12 +75,11 @@ describe('DynamicExtensionComponent', () => {
         });
 
         it('should load the TestComponent', () => {
-            const innerElement = fixture.debugElement.query(By.css('[data-automation-id="found-me"]'));
-            expect(innerElement).not.toBeNull();
+            expect(testingUtils.getByDataAutomationId('found-me')).not.toBeNull();
         });
 
         it('should pass through the data', () => {
-            const testComponent = fixture.debugElement.query(By.css('test-component')).componentInstance;
+            const testComponent = testingUtils.getByCSS('test-component').componentInstance;
```

Comment:
This can be improved even more - you can use getByDirective function from testingUtils and get that component by class name (better because we can avoid typos in future)

Reply:
I had to revert testUtlis from this file, as it produced circular dependancie

Reply:
can it be checked with @MichalKinas why it was causing circular dependency? Maybe it's something related with type of path, maybe we should not use @adf-core here?
<img width="1049" alt="image" src="https://github.com/user-attachments/assets/6cf48ffb-92c7-44eb-ac92-28ea2356bd6b" />

Reply:
@AleksanderSklorz , @MichalKinas  With local path I have a circular dependency as well: `extensions:build:production --> core:build:production --> extensions:build:production`

Reply:
ok, please check with Michał then if that is some known issue

Reply:
Yeah that error is correct, extension are the starting point and core is dependent on that, therefore we cannot make extensions dependent on anything from core